mlir/python/mlir/dialects/VectorOps.td

```diff
@@ -1,14 +1,14 @@
-//===-- VectorOps.td - Entry point for VectorOps bind ------*- tablegen -*-===//
+//===-- Vector.td - Entry point for Vector bindings --------*- tablegen -*-===//
 //
```

Comment:
Should the file be renamed to Vector.td as well?

Reply:
Vector.td now contains the dialect and gets included in VectorAttributes.td .

Reply:
I am sorry I misunderstood which file you were talking about .. debugging python as we speak, this may have to remain VectorOps actually.

Reply:
yeah, the python side actually wants the xxxOps.td file, leaving as is lets tests pass

Reply:
Not a python binding expert but if it is just exporting the ops then that sounds more reasonable yes!